pkg/dnsmanager/dns_manager.go

```diff
@@ -18,13 +17,23 @@ type DNSManager struct {
 var _ DNSManagerClient = (*DNSManager)(nil)
 var _ DNSResolver = (*DNSManager)(nil)
 
-func CreateDNSManager(ctx context.Context, cfg config.Config, k8sClient k8sclient.K8sClientInterface, storageClient storage.StorageClient, clusterName string) *DNSManager {
+func CreateDNSManager() *DNSManager {
 	return &DNSManager{}
 }
 
 func (dm *DNSManager) ProcessDNSEvent(dnsEvent tracerdnstype.Event) {
-	for _, address := range dnsEvent.Addresses {
-		dm.addressToDomainMap.Set(address, dnsEvent.DNSName)
+	if dnsEvent.NumAnswers > 0 {
+		// FIXME use dnsEvent.Nameserver to resolve the DNS name
+		addresses, err := net.LookupIP(dnsEvent.DNSName)
```

Comment:
we want to specifically use the ig dns resolution...

Reply:
why did we update the ig version?

Reply:
For a fix on arm processors... But are you sure it works with the older IG?

Reply:
the uncrompessed issue doesn't bother us since it's only inside the cluster

Reply:
yes I am sure. I will try to figure out what happened on the last version

Reply:
We should still have a workaround when there's no address